adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m

```diff
@@ -53,20 +56,30 @@ GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize) {
   return GADAdSizeInvalid;
 }
 
-BannerSize GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize) {
-  if (GADAdSizeEqualToSize(adSize, GADAdSizeMediumRectangle)) {
-    return BannerSizeMrec;
-  }
-  if (adSize.size.height == GADAdSizeLeaderboard.size.height) {
-    return BannerSizeLeaderboard;
-  }
+VungleAdSize *_Nonnull GADMAdapterVungleConvertGADAdSizeToBannerSize(GADAdSize adSize) {
```

Comment:
Not find the caller to this method, not used any more?

Reply:
yes not used. but shiv wanted us to leave it.